src/NServiceBus.Transport.SQS/IAmazonSqsMessageTranslationStrategy.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace NServiceBus.Transport.SQS
+{
+    using System.Collections.Generic;
+    using Amazon.SQS.Model;
+    using static TransportHeaders;
+
+    /// <summary>
+    /// Configures how the incoming SQS transport message is extracted
+    /// </summary>
+    public interface IAmazonSqsIncomingMessageExtractor
+    {
+        /// <summary>
+        /// Performs the SQS transport message extraction
+        /// </summary>
+        /// <param name="receivedMessage"></param>
+        /// <param name="messageId"></param>
+        /// <param name="headers"></param>
+        /// <param name="s3BodyKey"></param>
+        /// <param name="body"></param>
+        /// <returns></returns>
+        bool TryExtractMessage(Message receivedMessage, string messageId, out Dictionary<string, string> headers, out string s3BodyKey, out string body);
+    }
```

Comment:
this should go in a separate `IAmazonSqsIncomingMessageExtractor.cs` file

Reply:
Thanks - missed that after our initial run through - separated into 2 different files.